README.md

```diff
@@ -1,6 +1,12 @@
-# SpeedRunMod
-hello world
+# TerrariaSpeedrunVerificationMod
 
-Hi BlobDylan!
+Features:
+
+-by having the mod installed the user has a built in timer, when activated, a hash key will be set and displayed verifying the run. if the timer is stopped the key goes away, the key will also be somewhat transparent and change every once in a while making it harder to tamper with.
+
+-the user will also have the option to turn the timer display off incase they want to use a third party timer.(not sure about this one...)
```

Comment:
I agree there should be some type of "minimalist" option where either the timer is very small and/or almost completely transparent. This brings up an interesting question, though. Does this replace an already existing timer? I'm assuming this mod would be fine as the only source of timer, but something to think about

Reply:
Indeed it is. I think the timer should probably either be mandatory or non existant since having a timer there just as a tool for the user might be nice but its probably not needed. atleast for the 1st version of the mod.